scripts/install_third_party_libs.py

```diff
@@ -125,9 +125,25 @@ def pip_install(package, version, install_path):
 
     # The call to python -m is used to ensure that Python and Pip versions are
     # compatible.
-    subprocess.check_call([
-        sys.executable, '-m', 'pip', 'install', '%s==%s' % (package, version),
-        '--target', install_path])
+    command = [
+        sys.executable, '-m', 'pip', 'install', '%s==%s'
+        % (package, version), '--target', install_path]
+    process = subprocess.Popen(
+        command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    stdout, stderr = process.communicate()
+    if stderr == '':
```

Comment:
Rewrite as `if not stderr:`

Reply:
Done! Ptal
Also regarding the previous comment, I won't be able to edit labels and assign till I finish another PR, I will make sure I follow that when I finish my second PR.
Thanks for reviewing!